### PR TITLE
TST: Add more tests

### PR DESCRIPTION
**Pull request by ElDeveloper**
Covers:

- `get_user_for_kit`.
- `get_menu_items`.
- `check_if_consent_exists`.
- `get_user_info`.

:us:-gut

### CODE REVIEW

File: amgut/lib/data_access/test/test_ag_data_access.py [hunk 1]
```diff
@@ -272,16 +272,46 @@ def test_checkPrintResults_invalid_ids(self):
...
+        obs = self.ag_data.get_menu_items('tst_VpQsT')
+        self.assertEqual(({'REMOVED-0': []}, {}, [], True), obs)
+
+    def test_get_menu_items_errors(self):
+        with self.assertRaises(RuntimeError):
+            self.ag_data.get_menu_items('tst_esXXX')
 
     def test_check_if_consent_exists(self):
-        raise NotImplementedError()
+        obs = self.ag_data.check_if_consent_exists(
```
Comment (squirrelo) [inline amgut/lib/data_access/test/test_ag_data_access.py]: What happens if the ag_login_id does not exist?

Comment (squirrelo) [inline amgut/lib/data_access/test/test_ag_data_access.py]: To clarify, what if it does not exist in the database?

File: amgut/lib/data_access/test/test_ag_data_access.py [hunk 2]
```diff
@@ -272,16 +272,46 @@ def test_checkPrintResults_invalid_ids(self):
...
+            '00711b0a-67d6-0fed-e050-8a800c5d7570', 'REMOVED-42')
+        self.assertTrue(obs)
+
+        obs = self.ag_data.check_if_consent_exists(
+            '00711b0a-67d6-0fed-e050-8a800c5d7570', 'REMOVED-111')
+        self.assertFalse(obs)
 
     def test_get_user_info(self):
-        raise NotImplementedError()
+        obs = self.ag_data.get_user_info('tst_wAhSB')
```
Comment (squirrelo) [inline amgut/lib/data_access/test/test_ag_data_access.py]: Can you test getting user info for a non-existent kit id?

File: amgut/lib/data_access/test/test_ag_data_access.py [hunk 3]
```diff
@@ -272,16 +272,46 @@ def test_checkPrintResults_invalid_ids(self):
...
+        exp = {'address': 'REMOVED', 'ag_login_id':
+               'd8592c74-84a5-2135-e040-8a80115d6401', 'city': 'REMOVED',
+               'country': 'REMOVED', 'email': 'REMOVED', 'name': 'REMOVED',
+               'state': 'REMOVED', 'zip': 'REMOVED'}
+        self.assertEqual(exp, obs)
```
Comment (mortonjt) [inline amgut/lib/data_access/test/test_ag_data_access.py]: Would it worthwhile to have a test when the user isn't found in the database?

**Comment by ElDeveloper** [comment]
@mortonjt & @squirrelo, comments should be addressed.

File: amgut/lib/data_access/test/test_ag_data_access.py [hunk 4]
```diff
@@ -272,16 +272,51 @@ def test_checkPrintResults_invalid_ids(self):
...
     def test_get_user_for_kit(self):
-        raise NotImplementedError()
+        obs = self.ag_data.get_user_for_kit('tst_IueFX')
+        self.assertEqual('ded5101d-c8e3-f6b3-e040-8a80115d6f03', obs)
+
+        obs = self.ag_data.get_user_for_kit('tst_esABz')
+        self.assertEqual('d8592c74-8421-2135-e040-8a80115d6401', obs)
+
+    def test_get_user_for_kit_errors(self):
+        with self.assertRaises(RuntimeError):
```
Comment (wasade) [inline amgut/lib/data_access/test/test_ag_data_access.py]: should this be a `RuntimeError`?

**Pull request merged**